scripts/install.sh

```diff
@@ -108,7 +108,7 @@ function check_dist() {
 	dist=$(echo "${dist}" | tr '[:upper:]' '[:lower:]')
 
 	case "${dist}" in
-		debian|ubuntu|centos|fedora)
+		debian|ubuntu|centos|cloudlinux|fedora)
```

Comment:
Add another section that does `echo "WARN ${dist} is not officially supported. Attempting RPM install"` so that it's obvious that CloudLinux is not officially supported.

Reply:
Done ✅